src/Umbraco.Web/PublishedCache/NuCache/ITransactableDictionary.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Umbraco.Web.PublishedCache.NuCache
+{
+    public interface ITransactableDictionary<TKey, TValue> :
+        ITransactable,
+        CSharpTest.Net.Collections.IConcurrentDictionary<TKey, TValue>,
```

Comment:
I don't think we want to use an abstraction that is part of a library that isn't part of our Core. Ideally these interfaces will move to the Core project. Can we create our own abstraction if that's necessary?